dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnectorTest.java

```diff
@@ -34,6 +34,39 @@ public class AbstractOracleConnectorTest {
   private final ConnectorArguments arguments = mock(ConnectorArguments.class);
   private final AbstractOracleConnector connector = new OracleMetadataConnector();
 
+  @Test
+  public void getDefaultFileName_notForAssessment_success() {
+    OracleLogsConnector logs = new OracleLogsConnector();
```

Comment:
I would strongly suggest to put the test for `OracleLogsConnector` in the `OracleLogsConnectorTest` class. Similarly for other connectors and also other test methods that group different connectors in one test.

However, (in the context of this specific test method) if the method for generating the default filename is extracted to a utils class, then the situation would be simplified, because it would be enough to test this utils class only.

Reply:
Done, I've split the tests into testing on the enum and testing if connectors have the right enum. They're placed in the enum's and the connector subclasses' test classes respectively.